evaluation_script/main.py

```diff
@@ -1,8 +1,20 @@
 import random
-
+import logging
+import time
+import sys
 
 def evaluate(test_annotation_file, user_submission_file, phase_codename, **kwargs):
     print("Starting Evaluation.....")
+
+    eval_script_logger = logging.getLogger(name='eval_script')
+    eval_script_logger.setLevel(logging.DEBUG)
+
+    handler = logging.StreamHandler(sys.stdout)
+    handler.setLevel(logging.DEBUG)
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    handler.setFormatter(formatter)
+    eval_script_logger.addHandler(handler)
+
```

Comment:
can you create a new method called `get_logger` in this script and use that to get the logger here?